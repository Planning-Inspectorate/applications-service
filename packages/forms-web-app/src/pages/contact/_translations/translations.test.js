const enContactTranslations = require('./en.json');
const cyContactTranslations = require('./cy.json');

describe('pages/contact/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enContactTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyContactTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enContactTranslations).toHaveSameKeys(cyContactTranslations);
	});
});
