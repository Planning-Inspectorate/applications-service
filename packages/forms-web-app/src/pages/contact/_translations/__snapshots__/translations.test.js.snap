// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/contact/_translations should return the correct English translations 1`] = `
{
  "heading1": "Contact us",
  "heading2": "General Enquiries",
  "heading3": "Feedback and complaints",
  "heading4": "Press and media",
  "paragraph1": "If you are contacting us about a particular project, please email the project team. The project team email address can be found on the project page.",
  "paragraph10": "Further information about contacting the Planning Inpectorate customer services can be viewed on {{-link}}.",
  "paragraph11": "The Planning Inspectorate tries to provide the best possible service for its customers. We try to ensure everyone is satisfied with the service they receive. Complaints about the examination process or Examining Authority's final report will be considered and responded to.",
  "paragraph12": "Telephone:",
  "paragraph13": "Email:",
  "paragraph14": "The Press office is open from {{-openingHours}}.",
  "paragraph2": "If you have another enquiry, you can contact us by the following options:",
  "paragraph3": "Contact form: {{-link}}",
  "paragraph3LinkText": "Fill out our form",
  "paragraph4": "Email: {{-link}}",
  "paragraph5": "If you are contacting us about a specific project, put the project name in the subject line. At busy times we may not be able to respond immediately.",
  "paragraph6": "Telephone customer services:",
  "paragraph8": "Address:",
  "paragraph9": "Follow us on X:",
}
`;

exports[`pages/contact/_translations should return the correct Welsh translations 1`] = `
{
  "heading1": "Cysylltu â ni",
  "heading2": "Ymholiadau Cyffredinol",
  "heading3": "Adborth a chwynion",
  "heading4": "Y wasg a'r cyfryngau",
  "paragraph1": "Os ydych yn cysylltu â ni ynglŷn â phrosiect penodol, anfonwch neges e-bost at dîm y prosiect. Gellir gweld cyfeiriad e-bost tîm y prosiect ar dudalen y prosiect.",
  "paragraph10": "Gallwch weld rhagor o wybodaeth am gysylltu â gwasanaethau cwsmeriaid yr Arolygiaeth Gynllunio ar {{-link}}.",
  "paragraph11": "Mae'r Arolygiaeth Gynllunio yn ceisio darparu'r gwasanaeth gorau posibl i'w chwsmeriaid. Ceisiwn sicrhau bod pawb yn fodlon ar y gwasanaeth a gânt. Bydd cwynion ynglŷn â'r broses archwilio neu adroddiad terfynol yr Awdurdod Archwilio yn cael eu hystyried a rhoddir ymateb iddynt.",
  "paragraph12": "Ffôn:",
  "paragraph13": "E-bost:",
  "paragraph14": "Mae Swyddfa'r Wasg ar agor o {{-openingHours}}.",
  "paragraph2": "Os oes gennych ymholiad arall, gallwch gysylltu â ni trwy'r opsiynau canlynol:",
  "paragraph3": "Ffurflen gyswllt: {{-link}}",
  "paragraph3LinkText": "Llenwch ein ffurflen",
  "paragraph4": "E-bost: {{-link}}",
  "paragraph5": "Os ydych yn cysylltu â ni ynglŷn â phrosiect penodol, rhowch enw'r prosiect yn y llinell bwnc. Mae'n bosibl na fyddwn yn gallu ymateb ar unwaith ar adegau prysur. ",
  "paragraph6": "Ffoniwch y gwasanaethau cwsmeriaid:",
  "paragraph8": "Cyfeiriad:",
  "paragraph9": "Dilynwch ni ar X:",
}
`;
